src/nets/e2e_asr_attctc_th.py

```diff
@@ -1656,6 +1688,9 @@ def forward(self, hpad, hlen, ys):
         :return:
         '''
         hpad = mask_by_length(hpad, hlen, 0)
+        if not isinstance(hlen, list):
+            hlen = hlen.tolist()
```

Comment:
In the case of subsample, `hlen` can be list of tensor, right?
https://github.com/ShigekiKarita/espnet/blob/20ca6b814643ba8542dfae50f6cd318d89babeff/src/nets/e2e_asr_attctc_th.py#L2111
Therefore it is better to change as follows:
```python
    hlen = list(map(int, hlen))
```
But I’m not sure how-to-detect this case.

Reply:
Yeah, we may need test-case for this.

Reply:
I gonna change this https://github.com/ShigekiKarita/espnet/blob/20ca6b814643ba8542dfae50f6cd318d89babeff/src/nets/e2e_asr_attctc_th.py#L2111 being list of int